packages/fxa-auth-server/lib/senders/emails/partials/paymentProvider/en.ftl

```diff
@@ -2,6 +2,9 @@
 payment-method = Payment Method:
 payment-provider-paypal-plaintext = { payment-method } { -brand-paypal }
 # Variables:
-#  $cardType (String) - The type of the credit card, e.g. Visa
+#  $cardType (String) - The type of the credit card, all lowercase, e.g. amex
+#  $cardName (String) - The brand name of the credit card, e.g. American Express
 #  $lastFour (String) - The last four digits of the credit card, e.g. 5309
 card-ending-in = { $cardType } card ending in { $lastFour }
+cc-card-ending-in = { $cardName } card ending in { $lastFour }
```

Comment:
Should this be "credit-card-ending-in"?
Otherwise wouldn't "cc-card" be credit card card?